sig-docs/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website, infrastructure and tooling
+
+#### Cross-cutting and Externally Facing Processes
+
+- Standards for content: contribution and reviews
+- Coordinating docs contributions for quarterly releases
+
+### Out of scope
+
+- Primary documentation of new features
+
+## Roles and Organization Management
+
+This SIG adheres to the standards for roles and organization management as specified by [sig-governance]. This SIG opts in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Chairs
+
+Chairs also serve as Tech Leads.
+
+### Deviations from [sig-governance]
+
+Per [readme]: 
+
+- Meetings are weekly
+- Once per month, weekly meeting time changes for easier attendance from APAC contributors per
+
+### Subproject Creation
+
+SIG Chairs
```

Comment:
This isn't explicitly given as one of the options but just to be explicit I am in support of this, and read it as Option 1.

Reply:
Lets make sure it is clear.

Reply:
What do we need to clarify here?